lib/iris/tests/unit/fileformats/grib/load_convert/test_product_definition_template_31.py

```diff
@@ -22,25 +22,23 @@
 from __future__ import (absolute_import, division, print_function)
 from six.moves import (filter, input, map, range, zip)  # noqa
 
-# import iris tests first so that some things can be initialised
+# import iris.tests first so that some things can be initialised
 # before importing anything else.
 import iris.tests as tests
 
 from copy import deepcopy
-import warnings
-
+import mock
 import numpy as np
+import warnings
```

Comment:
Surely `warnings` is a builtin and therefore should be in the previous group of imports?